### ISSUE TITLE
Kotlin changes are treated like Java code

### ISSUE THREAD

**Issue by dimonchik0036** [JetBrains-Research/RefactorInsight#122]
The diff from the plugin:
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/22503910/224751829-092e0d05-9f4b-45b4-8e7e-640b3dddec44.png">

The original diff:
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/22503910/224751992-e646966f-9753-4c5e-82e3-98e995495863.png">

Even an icon came from Java:
<img width="585" alt="image" src="https://user-images.githubusercontent.com/22503910/224752543-775b05d2-5d0a-4ebc-8f35-e81410559f8b.png">

**Comment by dimonchik0036** [comment]
Common question: is this plugin supposed to work with Kotlin code? I see very often suggestions for Java code, but this is the first time I see something for Kotlin

**Comment by onewhl** [comment]
@dimonchik0036 the plugin works with both Java and Kotlin, but it detects much more refactorings in Java code than in Kotlin. We plan to increase the number of refactorings the plugin is able to detect in Kotlin.

In the case you described, the plugin correctly identified Rename method refactoring, right?

And thank you for using the plugin! If you have any suggestions on how we can improve it or what features would be useful for you, please let us know! :)

**Comment by dimonchik0036** [comment]
>In the case you described, the plugin correctly identified Rename method refactoring, right?

Yep. The diff is correct too. The issue is only about how it looks – it treats as Java (probably some misconfiguration bug)

**Comment by onewhl** [comment]
Hi @dimonchik0036, in the next release plugin will show icons depending on the file type (Kotlin icons for Kotlin files, Java icons for Java files). Thanks again for submitting the issue, I just didn't notice this before.
<img width="707" alt="Screen Shot 2023-03-24 at 5 22 26 PM" src="https://user-images.githubusercontent.com/10652414/227586035-7b0ba6cb-872a-4bb7-bac5-0bb24d9fcf81.png">

About the color scheme in the editor -- could you please clarify if it changes the color scheme every time you open a file in the changes browser? Or only when you open refactoring visualization by double-clicking on the item in the list (when R button is selected)?

**Comment by dimonchik0036** [comment]
Each language has its own color scheme. So Java files have one color, Kotlin files have another color. 
I think the problem is that you are using [JavaClassFileType](https://github.com/JetBrains-Research/RefactorInsight/blob/ecf26e64572ffa1f0934e94a6e4f63b64b5d9a8c/plugin/src/main/java/org/jetbrains/research/refactorinsight/ui/windows/DiffWindow.java#L119) everywhere instead of the original file type

**Comment by onewhl** [comment]
@dimonchik0036 Oh, I see, thank you!
